project/web/app.sls

```diff
@@ -29,6 +29,8 @@ gunicorn_conf:
         settings: "{{ pillar['project_name'] }}.settings.deploy"
         virtualenv_root: "{{ vars.venv_dir }}"
         directory: "{{ vars.source_dir }}"
+        newrelic_license_key: "{{ vars.newrelic_license_key }}"
```

Comment:
Will this work as expected if the key isn't set? I think it might set the Jinja variable `newrelic_license_key` to the string `"False"`, which would then evaluate to `True` in gunicorn.conf. (I haven't tested it yet, obviously)